src/iosrv/init.c

```diff
@@ -66,6 +67,9 @@ const char	       *dss_socket_dir = "/var/run/daos_server";
 /** NVMe shm_id for enabling SPDK multi-process mode */
 int			dss_nvme_shm_id = DAOS_NVME_SHMID_NONE;
 
+/** dRPC context */
+struct drpc	       *dss_drpc_ctx;
```

Comment:
Wondering if the dRPC stuff should be broken out into a different file. Are you adding an API to share this active dRPC context for other iosrv modules making drpc_calls as well?

Reply:
Where do you want the file to be? src/iosrv/drpc.c?

I'm not converting the other daos_io_server modules to use this context. But I think that's the way to go.

Reply:
How about src/iosrv/drpc_client.c? Since we also have a drpc server in the same directory (drpc_listener.c), it is good to make clear that this is acting as a drpc client.